.github/workflows/unit.yml

```
@@ -66,6 +66,9 @@ jobs:
               run: yarn test:types
             - name: Run unit tests
               run: yarn test:ci
+            # TODO [#4815]: enable all SSR v2 tests
+            - name: Run experimental SSR fixture tests
+              run: TEST_SSR_COMPILER=1 yarn test packages/@lwc/ssr-compiler/src/__tests__/fixtures.spec.ts
```

With #4809 this can just be:
```suggestion
              run: TEST_SSR_COMPILER=1 yarn test
```

---

This will still run every single test, which is unnecessary since the tests already ran in a previous step. 🙂